### PR TITLE
Uploaded user form for budget requirements in HTML

### PR DESCRIPTION
**Pull request by harjotb03**

### CODE REVIEW

**Review by amywangg — APPROVED** [review]
Good work harjot!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  <link rel="stylesheet" href="node_modules/bulma/css/bulma.css">
+  <link rel="stylesheet" href="css/mystyles.css">
+
+  <script defer src="https://use.fontawesome.com/releases/v5.14.0/js/all.js"></script>
+  <script src="javascript/script.js" charset="utf-8"></script>
+</head>
+
+<body>
+
+<!-- Reference to user name and welcome page -->
```
Comment (vivvvli) [inline index.html:20]: Good documentation/ commenting on what each piece of code means; really helped me to interpret what you did

File: index.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  </section>
+
+<!-- Preferences title line  -->
+  <div class="container">
+    <h1 class="is-size-3 has-text-weight-bold">Preferences</h1>
+  </div>
+
+<!-- Month input line -->
+  <section class="section">
+    <div class="container">
```
Comment (vivvvli) [inline index.html:34]: It seems like you were trying to apply multiple classes to one tag and ended up having one div tag for each; there might be a way to combine all of them together in one tag to make it cleaner?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
```
Comment (vivvvli) [inline index.html:1]: Great use of indentation and spacing throughout file! Especially helped with keeping track of tags 
All in all looks great :)

File: package.json [hunk 4]
```diff
@@ -1,40 +1,17 @@
...
-  },
+  "name": "mybulma",
+  "version": "1.0.0",
+  "description": "",
+  "main": "sass/mystyles.scss",
   "scripts": {
-    "start": "node server/server.js",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
```
Comment (amywangg) [inline package.json:23]: tip for next time! try not to remove things from package.json as other parts of the project are dependent on these packages :)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+          <div class="field is-narrow">
+            <div class="control">
+              <div class="select is-rounded is-medium">
+<!-- different dropdown options for category input  -->
+                <select>
+                  <option>Select Category</option>
+                  <option>Essential</option>
+                  <option>Non-Essential</option>
+                </select>
+              </div>
```
Comment (amywangg) [inline index.html:99]: Good use of html elements!

File: index.html [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+
+<!-- Monthly budget input line -->
+      <div class="field is-horizontal py-3">
+        <div class="field-label is-large">
```
Comment (aneesaali) [inline index.html:67]: i love the design of the CAD part! very nice touch

File: index.html [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+          <label class="label">Category</label>
+        </div>
+        <div class="field-body">
+          <div class="field is-narrow">
+            <div class="control">
+              <div class="select is-rounded is-medium">
+<!-- different dropdown options for category input  -->
+                <select>
+                  <option>Select Category</option>
```
Comment (aneesaali) [inline index.html:95]: for future, im not too sure if we need to have a category dropdown. i know our tool will be having manual category inputs so we might need to add an amount under the selected category
code is great! 👍  this comment is more for future reference when we implement this page onto our dashboard

**Pull request merged**